pkgs/by-name/kr/kryptor/update.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nix-shell
+#!nix-shell --pure -i bash -p bash nix nix-update git cacert
+set -euo pipefail
+
+nix-update kryptor
```

Comment:
If you like, you could use `"$UPDATE_NIX_ATTR_PATH"` here, and on the next line.

Some packages even do something like:

```
attr="${UPDATE_NIX_ATTR_PATH:-kryptor}"
```

to keep the script working when called directly.

It's only helpful if someone makes an overridden version of the package at another attribute path, so it's up to you whether you think it's worth it.

Reply:
I think it's cleaner as it is now, even if it won't work for overridden packages, thanks for the suggestion though.


Bit off topic but are there any plans for `passthru.updateScript = nix-update-script { }` supporting dotnet and updating `deps.nix`?
It works with for example pnpm packages, where it updates the hash inside `pnpmDeps = pnpm.fetchDeps { hash = "..."; ... }`

Also currently it's nice that each nuget package from `deps.nix` is a seperate derivation and can be built independently, but regarding to https://github.com/NixOS/nixpkgs/issues/327064, what do you think about something like `nugetDeps = dotnet_sdk.fetchNugetDeps { hash = "..."; ... };`? A similar movement is also happening for rust packages.

Reply:
> what do you think about something like nugetDeps = dotnet_sdk.fetchNugetDeps { hash = "..."; ... };?

The problem is that restore isn't deterministic.  There's no lock-file in most packages, and even if there was, the dotnet lockfile mechanism has some problems.

It's a good idea though.  Maybe we'll get there eventually.